src/fabric-1.19.2/procedures/parse_json.java copy.ftl

```diff
@@ -0,0 +1 @@
+new Gson().fromJson(${input$json}, com.google.gson.JsonObject.class)
```

Comment:
You should use a try-catch in case the provided string is not a valid JSON object, so the game will not crash (print the error in the LOGGER only). You can see an example for a getter here https://github.com/MCreator/MCreator/blob/master/plugins/generator-1.19.2/forge-1.19.2/procedures/entity_checkgamemode.java.ftl

(The same thing applies to other concerned blocks)

Reply:
You have duplicated it. Delete one as one of them doesn't have a good file naming (and by renaming it, you will have to remove the old one).